cli/src/main/java/io/github/bmarwell/jfmt/imports/NamedImportOrder.java

```diff
@@ -0,0 +1,25 @@
+package io.github.bmarwell.jfmt.imports;
+
+public enum NamedImportOrder {
+    // default is a reserved keyword, so we need to use "defaultorder" instead. :(
+    defaultorder("/io/github/bmarwell/jfmt/config/importorder.default.properties"),
+    equalsverifier("/io/github/bmarwell/jfmt/config/importorder.equalsverifier.properties"),
+    google("/io/github/bmarwell/jfmt/config/importorder.google.properties"),
+    eclipse("/io/github/bmarwell/jfmt/config/importorder.eclipse.properties"),
+    intellij("/io/github/bmarwell/jfmt/config/importorder.intellij.properties"),
+    enterprise("/io/github/bmarwell/jfmt/config/importorder.enterprise.properties");
+
+    private final String resourcePath;
+
+    NamedImportOrder(String resourcePath) {
+        this.resourcePath = resourcePath;
+    }
+
+    public String getResourcePath() {
+        return resourcePath;
+    }
+
```

Comment:
Keep in mind that I might just overlook something, but my understanding is, that 

    private final var x;
    public getX();

is more or less (method `Class::getX` would be missing) equivalent to `public final var x;`
and the letter should be preferred on account of being simpler and less code.

Am I oversimplifying this?

Reply:
You are not oversimplifying, but it is in this case just easier to resolve using valueOf() or using fromCli(). That would only be possible using a Map or Reflection otherwise (we are separating CLI names from implementations with parameters here).